queries.sql

```diff
@@ -0,0 +1,97 @@
+USE
+readme;
+
+-- Добавление типа поста-цитаты id 1
+INSERT INTO content_types (name, icon_class)
+VALUES ("post-quote", "post-quote");
+
+-- Добавление типа поста со ссылкой на вебсайт id 2
+INSERT INTO content_types (name, icon_class)
+VALUES ("post-link", "post-link");
+
+-- Добавление типа поста с изображением id 3
+INSERT INTO content_types (name, icon_class)
+VALUES ("post-photo", "post-photo");
+
+-- Добавление типа поста со ссылкой на видео id 4
+INSERT INTO content_types (name, icon_class)
+VALUES ("post-video", "post-video");
+
+-- Добавление типа поста с текстом id 5
+INSERT INTO content_types (name, icon_class)
+VALUES ("post-text", "post-text");
+
+-- Добавление тестового пользователя с email: test@test.com id 1
+INSERT INTO users (registered_at, email, login, password, avatar_path)
+VALUES ("2015-05-23T14:25:10", "test@test.com", "Лариса", "123456", "userpic-larisa-small.jpg");
```

Comment:
Для пользователей мы не храним пароли в открытом виде, поэтому предлагаю на текущем этапе использовать встроенную в. MYSQL хэш-функцию md5.

```suggestion
VALUES ("2015-05-23T14:25:10", "test@test.com", "Лариса", MD5("123456"), "userpic-larisa-small.jpg");
```